extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -402,10 +402,10 @@ static function isBlocked($text, $blockData, $writeStats = true) {
 	 * @author Maciej Błaszkowski <marooned at wikia-inc.com>
 	 * @author Władysław Bodzek
 	 */
-	static function findBlocked($text, $blocksData, $writeStats = true, &$matchingBlockData = null) {
+	static function findBlocked( $text, $blocksData, $writeStats = true, &$matchingBlockData = null ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "findBlocked" is 22 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)